Model/Api/UpdateCart.php

```diff
@@ -94,15 +94,10 @@ public function execute($cart, $add_items = null, $remove_items = null, $discoun
         try {
             $this->cartRequest = $cart;
             
-            // TODO the display_id will be updated in this PR https://github.com/BoltApp/bolt-magento2/pull/863, and we need to adjust per change
-            list($incrementId, $immutableQuoteId) = array_pad(
-                explode(' / ', $cart['display_id']),
-                2,
-                null
-            );
-            $parentQuoteId = $cart['order_reference'];
+            // Bolt server sends immutableQuoteId as order reference
+            $immutableQuoteId = $cart['order_reference'];
```

Comment:
@vitaliyreznikov Here the immutable quote id is read from the `order_reference` not `display_id` as stated in the PR description. It's confusing and looks wrong. Can you explain?

Reply:
> Idea is we can use metadata for all data, but not for cart id.
What prevents us to do that?

Reply:
Thank you Daniel, good catch. **order_reference** is correct one. I've updated PR description.